pkg/storaged/filesystem/utils.jsx

```diff
@@ -147,24 +147,35 @@ export const MountPoint = ({ fstab_config, forced_options, backing_block, conten
     }
 
     let extra_text = null;
-    if (!is_filesystem_mounted) {
+    if (client.in_anaconda_mode()) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19807-20240109-123450-67a2b589-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)